shared/libretime_shared/config/_models.py

```diff
@@ -197,6 +197,29 @@ class AudioOpus(BaseAudio):
     format: Literal[AudioFormat.OPUS] = AudioFormat.OPUS
 
 
+class HlsStream(BaseModel):
+    fragment_prefix: str = "mp3_128"
+    bitrate: str = "128k"
+    sample_rate: int = 44100
+    codec: str = "libmp3lame"
```

Comment:
Could we remove the defaults? It doesn't make sens to me, as we will have the default for a single quality, but not the other ones.